funread_backend/UserBadge/urls.py

```diff
@@ -0,0 +1,15 @@
+from UserBadge import views
+from django.urls import path
+from rest_framework.urlpatterns import format_suffix_patterns
+
+urlpatterns = [
+    path('api/userbadges/', views.award_badge_to_user),
+    path('api/userbadges/list/<int:user_id>/', views.list_user_badges),
+    path('api/userbadges/delete/<int:user_badge_id>/', views.delete_user_badge),
+     
+]
```

Comment:
Utilizar "-" para mejorar legibilidad. También aplicar las mejores prácticas listadas arriba.
```suggestion
urlpatterns = [
    path('api/user-badges/', views.award_badge_to_user),
    path('api/user-badges/user/<int:user_id>/', views.list_user_badges),
    path('api/user-badges/<int:user_badge_id>/', views.delete_user_badge),
     
]
```